### ISSUE TITLE
Can't train on multiple GPUs - segmentation error

### ISSUE THREAD

**Issue by fschvart** [OpenGVLab/InternImage#80]
Hi,
Thank you so much for your amazing work on this SOTA algorithm!

When I train a model using 1 GPU (python train.py ./config/myconfigfile.py the model starts training
However, when I try to use diet_train.sh to run training on my 2 GPUs, I get the following error:

2023-04-04 18:59:11,688 - mmseg - INFO - Set random seed to 122144372, deterministic: False
2023-04-04 18:59:11,689 - mmseg - INFO - using core type: DCNv3
2023-04-04 18:59:11,689 - mmseg - INFO - using activation layer: GELU
2023-04-04 18:59:11,689 - mmseg - INFO - using main norm layer: LN
2023-04-04 18:59:11,689 - mmseg - INFO - using dpr: linear, 0.5
2023-04-04 18:59:11,689 - mmseg - INFO - level2_post_norm: True
2023-04-04 18:59:11,689 - mmseg - INFO - level2_post_norm_block_ids: [5, 11, 17, 23, 29]
2023-04-04 18:59:11,689 - mmseg - INFO - res_post_norm: True
/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/mmseg/models/losses/cross_entropy_loss.py:235: UserWarning: Default ``avg_non_ignore`` is False, if you would like to ignore the certain label and average loss over non-ignore labels, which is the same with PyTorch official cross_entropy, set ``avg_non_ignore=True``.
  warnings.warn(
/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/mmseg/models/losses/cross_entropy_loss.py:235: UserWarning: Default ``avg_non_ignore`` is False, if you would like to ignore the certain label and average loss over non-ignore labels, which is the same with PyTorch official cross_entropy, set ``avg_non_ignore=True``.
  warnings.warn(
2023-04-04 18:59:26,595 - mmseg - INFO - load checkpoint from http path: https://huggingface.co/OpenGVLab/InternImage/resolve/main/internimage_h_jointto22k_384.pth
WARNING:torch.distributed.elastic.multiprocessing.api:Sending process 28789 closing signal SIGTERM
ERROR:torch.distributed.elastic.multiprocessing.api:failed (exitcode: -9) local_rank: 0 (pid: 28788) of binary: /home/fabian/miniconda3/envs/intern/bin/python
Fatal Python error: Segmentation fault

Current thread 0x00007f6e9a339280 (most recent call first):
  File "/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/torch/distributed/elastic/utils/store.py", line 31 in get_all
  File "/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/torch/distributed/elastic/utils/store.py", line 53 in synchronize
  File "/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/torch/distributed/elastic/utils/store.py", line 67 in barrier
  File "/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 906 in _exit_barrier
  File "/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 877 in _invoke_run
  File "/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/torch/distributed/elastic/agent/server/api.py", line 709 in run
  File "/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/torch/distributed/elastic/metrics/api.py", line 125 in wrapper
  File "/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 236 in launch_agent
  File "/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/torch/distributed/launcher/api.py", line 131 in __call__
  File "/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/torch/distributed/run.py", line 715 in run
  File "/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/torch/distributed/run.py", line 724 in main
  File "/home/fabian/miniconda3/envs/intern/lib/python3.10/site-packages/torch/distributed/elastic/multiprocessing/errors/__init__.py", line 345 in wrapper
  File "/home/fabian/miniconda3/envs/intern/bin/torchrun", line 8 in <module>

Extension modules: torch._C, torch._C._fft, torch._C._linalg, torch._C._nn, torch._C._sparse, torch._C._special, numpy.core._multiarray_umath, numpy.core._multiarray_tests, numpy.linalg._umath_linalg, numpy.fft._pocketfft_internal, numpy.random._common, numpy.random.bit_generator, numpy.random._bounded_integers, numpy.random._mt19937, numpy.random.mtrand, numpy.random._philox, numpy.random._pcg64, numpy.random._sfc64, numpy.random._generator (total: 19)
Segmentation fault

I tried using torchrun and got the same error.

Any idea what might cause it? I'm using CUDA 11.3 + PyTorch 1.11 in WSL
Thanks
